pkg/apis/forklift/v1alpha1/provider.go

```diff
@@ -42,15 +42,6 @@ const (
 	Token = "token"
 )
 
-//
-// Provider pair.
-type ProviderPair struct {
```

Comment:
Removing this because it does not add value.  Better to just use provider.Pair instead.